### ISSUE TITLE
Calendar slow to load

### ISSUE THREAD

**Issue by brentdur** [TechAtNYU/cal#22]
perhaps considering loading part of the data at a time? Rather than everything.

I feel like it would be better to have a small load every two months (how many users are curious about +/- two months) then have a huge load at the beginning, which encourages users to leave

**Comment by AbhiAgarwal** [comment]
Perhaps we can start having pagination on the API or have a `?limit=100` type of parameter. Unless we already have this (@ethanresnick?). We can probably just load a subsection of the events data on load, and then load more if the user goes back more than a couple months?

**Comment by ethanresnick** [comment]
Yeah, we should definitely have pagination, and we don't now (except at one endpoint, I think). There's [an open issue](https://github.com/ethanresnick/json-api/issues/89) for it. Maybe we can figure out how to implement it efficiently when I'm next in NYC.
